### ISSUE TITLE
Python Errors

### ISSUE THREAD

**Issue by platinumvoid** [CodeXTF2/Burp2Malleable#5]
There seem to be issues in the following lines of code in lines 114-116:
```
       for x in resheaders.keys():
            printmsg(f"{x}: {reqheaders[x]}")
        headername = input("Header name: ")
        if headername in resheaderlist:
```

The reqheaders generates an error KeyError: 'Pragma'
The resheaderlist variable generates an error NameError: name 'resheaderlist' is not defined. Did you mean: 'reqheaderlist'?

To replicate the above select header placement for all parameters. All headers selected were new headers!

**Comment by CodeXTF2** [comment]
will take a look at this in a few days when my exams are over lol

**Comment by CodeXTF2** [comment]
fixed in 4da348eb80d9526ddb6a400446bff4a1819849a1. Feel free to test and reopen if issue is still present.
